src/js/tech/ChromecastTech.js

```diff
@@ -651,7 +653,19 @@ ChromecastTech = {
    _listenToPlayerControllerEvents: function() {
       var eventTypes = cast.framework.RemotePlayerEventType;
 
+      this._addEventListener(this._remotePlayerController, eventTypes.MEDIA_INFO_CHANGED, () => {
+         if (typeof this._getMediaSession().activeTrackIds[0] === 'number') {
+            const id = this._getMediaSession().activeTrackIds[0];
 
+            const tracks = this._getMediaSession().media.tracks;
+
+            const label = tracks[id].name;
+
+            this._onChangeSubtitleTrack(label);
```

Comment:
Pass an object (either the whole `track` or just `{ name }`)

Reply:
and what do we pass when there are no subtitles 
`{name: 'off'}` ?